src/relative-time.ts

```diff
@@ -10,16 +13,25 @@ export default class RelativeTime {
     this.locale = locale
   }
 
-  toString(format?: string): string {
-    const ago = this.timeElapsed()
-    if (ago) {
-      return ago
+  toString({format = 'auto', tense = 'auto'}: {format?: Format; tense?: Tense} = {}): string | undefined {
+    const micro = format === 'micro'
+    if (tense === 'past') {
+      return micro ? this.microTimeAgo() : this.timeAgo()
+    }
+    if (tense === 'future') {
+      return micro ? this.microTimeUntil() : this.timeUntil()
     }
-    const ahead = this.timeAhead()
-    if (ahead) {
-      return ahead
+    if (format === 'auto') {
+      const ago = micro ? this.microTimeAgo() : this.timeElapsed()
```

Comment:
if `format` is already auto, `micro` would never be true here right?

Reply:
Nice catch!